script/swarm.mjs

```diff
@@ -66,7 +66,6 @@ function getHealthEstimate(token){
     case 'pf1':
     case 'pf2e':
     case 'dnd5e':
-      return token.actor.data.data.attributes.hp.value / token.actor.data.data.attributes.hp.max;
```

Comment:
`actor.data` was deprecated in v10 and replaced by `actor.system`.  Removing this line (and falling back to `actor.system` removes deprecation warnings being shown.